### PR TITLE
resolved long link horizontal scrolling problem

### PR DESCRIPTION
**Pull request by vkrm-segta**

### CODE REVIEW

File: packages/ui/src/views/chatmessage/ChatMessage.css [hunk 1]
```diff
@@ -75,6 +75,10 @@
 }
 
 .markdownanswer a {
+    display: block;
+    margin-right: 2.5rem;
+    word-wrap: break-word;
+    overflow-wrap: break-word;
```
Comment (HenryHengZJ) [inline packages/ui/src/views/chatmessage/ChatMessage.css:81]: dont think u need `overflow-wrap`.

somehow its still showing a little space there: 
image

Comment (vkrm-segta) [reply packages/ui/src/views/chatmessage/ChatMessage.css:81]: Sure working on it

Comment (vkrm-segta) [reply packages/ui/src/views/chatmessage/ChatMessage.css:81]: @HenryHengZJ 
While prompting if i get Request failed. I also need to move the horizontal scroll bar to see the full error.

https://github.com/FlowiseAI/Flowise/assets/124426279/ce4d5b4a-7e3f-430b-a643-dbaf7ce278c3

Comment (HenryHengZJ) [reply packages/ui/src/views/chatmessage/ChatMessage.css]: thanks for finding that out too! yea you can disable horizontal scrolling

**Pull request closed without merging**